src/commands/clusters/create.ts

```diff
@@ -104,6 +104,8 @@ export default class ClusterCreate extends BaseCommand {
   }
 
   private async createCluster() {
+    await ClusterUtils.checkClientVersion();
```

Comment:
Did not catch this the first time. This will not work. This will use just use the default context and see if that passes, we need to wait for the user to select which context to use and then check. Also if they specify a different kubeconfig then this also does not work.

Instead, this needs to be moved down to after.
```
const kube_contexts = await this.setupKubeContext(flags);
```